src/client/pages/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react'
+import { useTextKeys } from 'utils/textKeys'
 import { CheckoutPageWrapper } from 'components/checkout/CheckoutPageWrapper'
+import { Footer } from 'components/checkout/Footer'
 
 export const CheckoutPayment = () => {
-  return <CheckoutPageWrapper>{}</CheckoutPageWrapper>
+  const textKeys = useTextKeys()
+
+  return (
+    <CheckoutPageWrapper>
+      <Footer
+        buttonText={textKeys.CHECKOUT_FOOTER_COMPLETE_PURCHASE()}
+        buttonOnClick={() => {
+          console.log('click')
+        }}
```

Comment:
I'm guessing this is just a placeholder for later 😅

Reply:
No it's what should happen 🤪 (Yes, it's a placeholder)